### ISSUE TITLE
Clicking "outside" the time selector in the explore view should hide it

### ISSUE THREAD

**Issue by tomwilkie** [grafana/grafana#15228]
If you click the time selector in the explore view, the only way to hide it is to select a time or click the button again.  Clicking "outside" the time selector should hide it too.

Grafana v6.0.0-prebeta2 (eb8dfef)

**Comment by torkelo** [comment]
Agree, it does in dashboard but explore uses it's own copy for some reason.
